tabs/pid_tuning.js

```diff
@@ -8,9 +8,7 @@ var SUPEREXPO_FEATURE_BIT = 23;
 
 TABS.pid_tuning.initialize = function (callback) {
     var self = this;
-    if (GUI.active_tab != 'pid_tuning') {
-        GUI.active_tab = 'pid_tuning';
-    }
+    if (GUI.active_tab != 'pid_tuning') { GUI.active_tab = 'pid_tuning'; }
 
```

Comment:
As I tried to express before, it is not so much about what every one of the dozens of devs working on this is used or conditioned for, it is about everybody doing it the same way for this project. Otherwise we will end up with dozens of different fomattings within each individual file, which will make working with the code very hard.

Re the condition not being necessary. I agree with you that I can't see why it is needed, but there might be a not so obvious reason, and every tab uses a condition of this form. So the right thing is to either leave it in for this tab, or to remove it for all tabs, but **not** to remove it for one tab only. This goes under the 'don't leave it more messy than how it was when you found it' rule.